docs/topics/documenting/changelog.md

```diff
@@ -0,0 +1,37 @@
+# Keeping a changelog
+
+Your Saltext project includes scaffolding for keeping a changelog via [towncrier](https://towncrier.readthedocs.io/en/stable/).
+
+This changelog is rendered as part of the documentation.
+
+If you selected {question}`deploy_docs` == `rolling`, this changelog contains
+upcoming changes from the `main` branch on top of the crystallized changes
+from previous releases.
```

Comment:
I think some clarification is needed on how to turn the individual `*.md` pieces into the actual CHANGELOG.md when making a release.